### DISCUSSION TITLE
Is it possible to Display Sheet Music Only? (No Tabs?)

### DISCUSSION THREAD

**Discussion by arxtuvy-summer** [CoderLine/alphaTab#1574] [Q&A] [answered]
Hi everybody,

Do any of you know if it's possible to display sheet music only without the tablature using AlphaTab? I would like to use AlphaTab for non-fretted instruments, and was wondering if it's possible. If any of you know if this is possible, do you know what are the relevant parts of the code for controlling whether it's sheet music and/or tabs that get displayed?

Thank you, alphaTab community!

**Comment by Danielku15** [comment, marked as answer]
Yes this is all possible. 

https://www.alphatab.net/docs/showcase/special-tracks

Depending on the file formats it is encoded already there what "staves" to show. e.g. in Guitar Pro files it is specified on each track which staves to show. For non-fretted instruments this is configured accordingly. 
![image](https://github.com/user-attachments/assets/87a248df-800a-40bf-9637-70bfc33a41b6)

In MusicXML it depends on whether tab information is contained. 

alphaTab can be forced to not show tabs in two ways: 

1. You specify a stave profile in the settings which will hide tabs in all cases: https://www.alphatab.net/docs/reference/settings/display/staveprofile/
2. Before rendering, you manually adjust the staves to show in the data model.
    * [event for adjusting the data model](https://www.alphatab.net/docs/reference/api/scoreloaded) 
    * [the properties on the staff level to indicate how it should be visualized](https://github.com/CoderLine/alphaTab/blob/develop/src/model/Staff.ts#L80-L98).
